docs/en/FAQ/Protoc-Plugin-Fails-When-Build.md

```diff
@@ -0,0 +1,8 @@
+**Problem**：
+- When the Collector starts, it loads the protoc-plugins to produce the error “No such file or directory”.
```

Comment:
I doubt this is happening when collector starts, I think this is in the process of maven build.

Reply:
Ok, that's changed.